src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -42,6 +44,35 @@ class Lambda {
    * @return whether a piece of content was purged or not
    */
   private def sendPurgeRequest(contentId: String): Boolean = {
+    val request = basicPurgeRequest(contentId).build()
+
```

Comment:
tbh I'd have just added a parameter to sendPurgeRequest (..., soft: Boolean) and not doubled up on code